aerospike-core/src/query/filter.rs

```diff
@@ -185,7 +185,8 @@ macro_rules! as_within_radius {
             "{{ \"type\": \"AeroCircle\", \"coordinates\": [[{:.8}, {:.8}], {}] }}",
             lng, lat, radius
         );
-        let geo_string = as_val!(geo_json);
+        let geo_copy = geo_json.clone();
```

Comment:
I still do not understand why this is needed.

Reply:
I have removed creating the extra variable to hold the string but the reason we need to preserve the string is because of the wire protocol. The server expects the Filter to have Value [begin, end] of type string and not GeoJSON.